"form-control"}

```diff
@@ -0,0 +1,2 @@
+[log_symptom_update 9d86f26] Update no_when? method to check params[:when] == {:class=
+ 3 files changed, 4 insertions(+), 5 deletions(-)
```

Comment:
what's this file??

Reply:
that's what i was wondering. i think it's one of those commit files that comes up in the editor sometimes.. i just deleted it off this branch and will update the PR since jessye went to bed :)